livekit-agents/livekit/agents/voice_assistant/assistant.py

```diff
@@ -175,8 +175,8 @@ def on(self, event: EventTypes, callback: Callable | None = None) -> Callable:
                 - user_speech_committed: the user speech was committed to the chat context
                 - agent_speech_committed: the agent speech was committed to the chat context
                 - agent_speech_interrupted: the agent speech was interrupted
-                - function_calls_completed: all function calls have been completed
-                - will_synthesize_llm: the assistant will synthesize the LLM output
+                - function_calls_collected: all function calls have been completed
```

Comment:
TBH `collected` does not sound as clear as `completed`

Reply:
function_calls_finished means that all functions have been called as well as their execution.
function_calls_collected means that we just got all functions we need to launch

Reply:
got it.. in that case can we update the comment here? it still says: `all function calls have been completed`